test/distributed/test_jit_c10d.py

```diff
@@ -144,6 +144,36 @@ def forward(self, input: torch.Tensor):
         self.checkModule(TestModule(pg_nccl), (torch.rand((2, 3)),))
 
 
+class StoreTest(JitTestCase):
+    def setUp(self):
+        super(StoreTest, self).setUp()
+        self.file = tempfile.NamedTemporaryFile(delete=False)
+        self.filestore = torch.classes.dist_c10d.FileStore(self.file.name, 1)
+        self.prefix = "test_prefix"
+
+    def test_create_file_store(self):
+        # test FileStore creation in JIT
+        @torch.jit.script
+        def create_file_store(
+            path: str,
+            num_workers: int
+        ) -> torch.classes.dist_c10d.FileStore:
```

Comment:
Are there cases where we need this in eager? if so, maybe cover that as well?

Reply:
Yes actually the creation logic in eager already been covered in `setUp` and been tested in the `test_create_prefix_store`